docs/examples/cas9_kymotracking/cas9_kymotracking.rst

```diff
@@ -150,11 +157,11 @@ the longest line we found, and have a look at its position over time::
 
 .. image:: kymo_position_over_time.png
 
-We can use such a line to sample the photon counts in the image. If we want to sum the photon count in a pixel region
-around the line from -3 to 3, we can achieve this by::
+We can use the line coordinates to sample the photon counts in the image. If we want to sum the photon count in a pixel region
+around the line from -3 to 3 (a line with a width of 7 pixels) and subtract the background, we can achieve this by::
 
     plt.figure()
-    plt.plot(longest_line.seconds, longest_line.sample_from_image(3))
+    plt.plot(longest_line.seconds, longest_line.sample_from_image(3)-7*green_background_ppixel)
```

Comment:
One could consider making the effect of the window width explicit by doing something like
```python
window_size = 3
plt.plot(
    longest_line.seconds,
    longest_line.sample_from_image(window_size) - (2 * window_size + 1) * green_background_ppixel,
)
```
But then it might make sense to first assign the sampled array to not have the actual plot statement be so long.
```python
window = 3
bg_corrected = longest_line.sample_from_image(window) - (2 * window + 1) * green_background_pixel
plt.plot(longest_line.seconds, bg_corrected)
```